src/CMakeLists.txt

```diff
@@ -85,10 +85,11 @@ if(NOT IOS)
   add_subdirectory(simplewallet)
 endif()
 
-if(CMAKE_BUILD_TYPE STREQUAL Debug)
-  if (NOT BUILD_INTEGRATION)
-    add_subdirectory(debug_utilities)
-  endif()
+if(CMAKE_BUILD_TYPE STREQUAL Debug AND NOT BUILD_INTEGRATION)
+  message(STATUS "Building debug utilities")
+  add_subdirectory(debug_utilities)
+else()
+  message(STATUS "Not building debug utilities")
```

Comment:
This shouldn't depend on the build type: CMAKE_BUILD_TYPE is to control *how* you build, not *what* you build.
A couple alternatives:
- use an `option(BUILD_DEBUG_UTILS "Build debug utils" OFF)`
- always include the subdirectory, but then throw `EXCLUDE_FROM_ALL` on the `add_executable` calls so that they don't get built at all unless you explicitly run `make cn_deserialize` or `make object_sizes` from the cmake build dir.

Reply:
Added option(BUILD_DEBUG_UTILS ...)

Reply:
Added option(BUILD_DEBUG_UTILS ...)